apps/aeprimop/src/aeprimop.erl

```diff
@@ -215,7 +216,7 @@ oracle_extend_tx_instructions(Pubkey, DeltaTTL, Fee, Nonce) ->
 oracle_query_tx_instructions(OraclePubkey, SenderPubkey, Query,
                              QueryFee, QTTL, RTTL, TxFee, Nonce) ->
     [ force_inc_account_nonce_op(SenderPubkey, Nonce)
-    , spend_fee_op(SenderPubkey, TxFee + QueryFee)
+    , spend_fee_op(SenderPubkey, TxFee, QueryFee)
```

Comment:
Ah, Query fee still has to be paid by inner Tx

Reply:
Yes, the `QueryFee` does not go to the miner so it should be payed by the TX-actor.